recOrder/compute/qlipp_compute.py

```diff
@@ -207,27 +215,41 @@ def initialize_reconstructor(
 
     print("Initializing Reconstructor...")
     start_time = time.time()
-    recon = waveorder_microscopy(
-        img_dim=image_dim,
-        lambda_illu=lambda_illu,
-        ps=ps,
-        NA_obj=NA_obj,
-        NA_illu=NA_illu,
-        z_defocus=z_defocus,
-        chi=swing,
-        n_media=n_obj_media,
-        cali=cali,
-        bg_option=bg_correction,
-        A_matrix=inst_mat,
-        QLIPP_birefringence_only=anisotropy_only,
-        pad_z=pad_z,
-        phase_deconv=mode,
-        illu_mode="BF",
-        use_gpu=use_gpu,
-        gpu_id=gpu_id,
-    )
+    if pipeline != "fluorescence":
+        recon = waveorder_microscopy(
+            img_dim=image_dim,
+            lambda_illu=lambda_illu,
+            ps=ps,
+            NA_obj=NA_obj,
+            NA_illu=NA_illu,
+            z_defocus=z_defocus,
+            chi=swing,
+            n_media=n_obj_media,
+            cali=cali,
+            bg_option=bg_correction,
+            A_matrix=inst_mat,
+            QLIPP_birefringence_only=anisotropy_only,
+            pad_z=pad_z,
+            phase_deconv=mode,
+            illu_mode="BF",
+            use_gpu=use_gpu,
+            gpu_id=gpu_id,
+        )
+        recon.N_channel = n_channel
+    else:
```

Comment:
I don't have a better solution just yet, but fluorescence should be moved out of `qlipp_compute` in the future.